src/main/scala/codacy/scalastyle/ScalaStyle.scala

```diff
@@ -54,7 +54,7 @@ object ScalaStyle extends Tool {
   private lazy val nativeConfigFileNames = Set(configFileName, "scalastyle-config.xml")
 
   private lazy val defaultConfigInputStream: java.io.InputStream =
-    better.files.Resource.getAsStream(ScalastyleConfiguration.DefaultConfiguration)
+    better.files.Resource.getAsStream(ScalastyleConfiguration.DefaultConfiguration.stripPrefix("/"))
```

Comment:
Why do we need this? :o

Reply:
Reverted. ( It was because reading the resource with heading `/` doesn't work from a different jar )